samples/sqlservices/src/controllers/mainController.ts

```diff
@@ -701,9 +708,139 @@ export default class MainController implements vscode.Disposable {
 		await view.initializeModel(formWrapper);
 	}
 
-	//#endregion
+	private async getGraphTabContent(view: azdata.ModelView): Promise<void> {
+		const barChart = view.modelBuilder.chart<azdata.BarChartConfiguration>()
+			.withProps({
+				chartId: 'barChart1',
+				chartType: 'bar',
+				configuration: chartExamples.barConfig,
+				width: '500px',
+				height: '300px'
+			}).component();
+
+		const horizontalBarChart = view.modelBuilder.chart<azdata.BarChartConfiguration>() // BarChartConfiguration is used for both vertical and horizontal bar charts
+			.withProps({
+				chartId: 'horizBarChart1',
+				chartType: 'horizontalBar',
+				configuration: chartExamples.horizontalBarConfig,
+				width: '500px',
+				height: '300px'
+			}).component();
+
+		const lineChart = view.modelBuilder.chart<azdata.LineChartConfiguration>()
+			.withProps({
+				chartId: 'lineChart1',
+				chartType: 'line',
+				configuration: chartExamples.lineConfig,
+				width: '500px',
+				height: '300px'
+			}).component();
+
+		const pieChart = view.modelBuilder.chart<azdata.PieChartConfiguration>()
+			.withProps({
+				chartId: 'pieChart1',
+				chartType: 'pie',
+				configuration: chartExamples.pieConfig,
+				width: '300px',
+				height: '300px'
+			}).component();
+
+		const doughnutChart = view.modelBuilder.chart<azdata.PieChartConfiguration>() // PieChartConfiguration is used for both pie charts and doughnut charts
+			.withProps({
+				chartId: 'doughnutChart1',
+				chartType: 'doughnut',
```

Comment:
there was a comment in the original PR about using `donut` instead of `doughnut` since that's more commonly used for spelling charts. Was it decided not to switch?

Reply:
"Doughnut" is the proper spelling, "donut" is a colloquial misspelling, and I will die on this hill. 😆 Wikipedia redirects "donut" to "doughnut" and Excel uses "doughnut".  Chart.js uses "doughnut", though Highcharts uses "donut".

Reply:
THEY'RE ALL WRONG THEN

![image](https://github.com/microsoft/azuredatastudio/assets/28519865/7bd84d08-e19a-4215-88df-706f3a60e20c)

(/s)

Reply:
@Charles-Gagnon All I'm sayin' is that even the company-formerly-known-as-Dunkin-Donuts admitted the errors of their ways and rebranded to just _Dunkin'_ in 2019. 😛